OrbitGl/CaptureSerializer.cpp

```diff
@@ -240,17 +248,27 @@ ErrorMessageOr<void> CaptureSerializer::Load(std::istream& stream) {
     ERROR("%s", error_message);
     return ErrorMessage(error_message);
   }
-  ProcessCaptureData(capture_info);
+  CaptureData capture_data = GenerateCaptureData(capture_info);
 
   // Timers
   TimerInfo timer_info;
   while (ReadMessage(&timer_info, &coded_input)) {
-    time_graph_->ProcessTimer(timer_info);
+    if (timer_info.function_address() > 0) {
+      FunctionInfo* func =
+          capture_data.process()->GetFunctionFromAddress(timer_info.function_address());
+      CHECK(func != nullptr);
```

Comment:
This actually does not work. I just played around a little bit and loading a capture from file crashes here

Reply:
We don't store process in capture, so func should be nullptr here. Probably, we want to check if it's selected function instead and call `UpdateFunctionStats` only for this case and then call `time_graph_->ProcessTimer(timer_info, nullptr);` after "if"?

Reply:
Thanks for spotting this. I guess we want to find the function info here from the selected functions rather than from the process.

Reply:
And why do we need it at all?  I mean we store stats in proto, so we can remove one of those (stats from proto or UpdateFunctionStats from here).

Reply:
Wow, looked too long on this today. Right. We just need to call process timer here and do not update the stats. That actually might be a bug that existed before (where stats would be updated twice).

Reply:
But we should try to retrieve the function anyways, as in case of an orbit function, it is processed differently.

Reply:
I think that as function was nullptr this part (with updating stats) was just skipped